Dockerfile

```diff
@@ -1,7 +1,9 @@
 # ========== BUILD CONTAINER ===========
-FROM ubuntu:18.04 as conjur-cli-builder
+FROM ubuntu:21.10 as conjur-cli-builder
```

Comment:
just one question. did you manage to compare the build time between this version and the old one (before upgrating the ubuntu version?)

Reply:
No